sourmash/__main__.py

```diff
@@ -35,9 +36,10 @@
 categorize              Identify best matches for many signatures using an SBT.
 watch                   Classify a stream of sequences.
 
-** Other information:
+** Other commands:
 
-info                        Sourmash version and other information.
+info                        Display sourmash version and other information.
```

Comment:
we could go with `describe` instead of `info` - what do you think?

Reply:
+0

Reply:
I'm not really a fan of `info` for sourmash version and other info (because I always try `sourmash -v` for that).

I was going to say I like `describe` because it mirrors pandas, but pandas has both [`info`][0] and [`describe`][1] and this functionality matches `info` better...

[0]: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.info.html
[1]: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.describe.html

So... I think we keep `sourmash info` and `sourmash sig describe`, which doesn't make me happy but sounds less confusing than having `sourmash info` and `sourmash sig info`

Reply:
done in 18b2f18